### DISCUSSION TITLE
请问如何在console中切换av号转换功能？

### DISCUSSION THREAD

**Discussion by oscarwty** [the1812/Bilibili-Evolved#1545] [提问 / Q&A] [answered]
旧版本升级到v1.11.17后，无法再从面板关闭av号转换
https://github.com/the1812/Bilibili-Evolved/issues/1254#issuecomment-748575129
在这里看到，现在开关隐藏但状态保留，只能在console中切换，请问具体如何操作？

**Comment by CKylinMC** [comment]
虽然我不知道console那边的命令，我有一个取巧的办法。
首先导出设置，然后打开导出的那个json文件，找到下面这一串


```json
"bvidConvert":true
```

还有

```json
"preferAvUrl":true
```

true是开，false是关，改了再导入回去应该就行了。

_没测试过，但是理论可行.jpg_

**Comment by LonelySteve** [comment, marked as answer]
下面的方法是针对 Chrome 浏览器，其他浏览器应该也差不多：

1. 按 <kbd>Ctrl + Shift + I</kbd> 或者 <kbd>F12</kbd>，打开开发者调试工具
2. 按 <kbd>Ctrl + \`</kbd> 或者切换到 `Console` 选项卡，输入以下代码执行：

    ```
    bilibiliEvolved.settings.bvidConvert = false;
    bilibiliEvolved.settings.preferAvUrl = false;
    ```
    
    ![这图可能加载不出来（](https://user-images.githubusercontent.com/34429322/107520408-5b1ccc80-6bec-11eb-872c-26b7a6f1cd45.png)

其他的配置项大同小异，正如 @CKylinMC  所说，如果不清楚的话，可以导出设置文件参考一下，如果记住了配置名称，在控制台设置会更方便一些，至少还有一定的补全功能。

**Comment by Geeyun-JY3** [comment]
[v1.11.12](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.12) 的 release note 里有写：
> 原先开启的用户依然有效, 并且仍然可在浏览器控制台中修改设置
`bilibiliEvolved.settings.preferAvUrl = false`
